### PR TITLE
Cloudanix Update

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR Contains adds the following.
- Support for Kubernetes 1.25 and higher
- Spectro Namespace Labeler 
- yaml file format example

💻 [Cloudanix](https://deploy-preview-1276--docs-spectrocloud.netlify.app/integrations/cloudanix)

🎫 [Jira Ticket]()

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1276#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/06-integrations/00-cloudanix.md [hunk 1]
```diff
@@ -17,19 +17,19 @@ import Tooltip from "shared/components/ui/Tooltip";
...
-<br/>
+<br />
 
-- Detects files added or modified in sensitive directories.
-- SSH into a container. 
-- Modifications to shell configuration files.
-- Attempts to read sensitive files that contain credential information.
-- Crypto mining detection.
+- Files added or modified in sensitive directories
+- SSH into a container
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-cloudanix.md:25]: Let's space these out so it looks less crammed.

File: content/docs/06-integrations/00-cloudanix.md [hunk 2]
```diff
@@ -41,10 +41,10 @@ Additionally, you can start Jira workflows and target specific workloads from th
 
 - CPUs: 0.5
 - Memory: 256 MiB
-- Kubernetes 1.19.x to 1.24.x
+- Kubernetes 1.19.x to 1.25.x
 - Kernel version 4.5 and higher
 
-## Parameters
+## Parameters:
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-cloudanix.md:47]:
```suggestion
## Parameters
```
We don't use the `:`

File: content/docs/06-integrations/00-cloudanix.md [hunk 3]
```diff
@@ -70,12 +70,42 @@ This Helm Chart installs four Cloudanix services to enable container security ca
...
-- **threat-service**: A threat service that exports threat events and affected Kubernetes resources which are visible on the Cloudanix dashboard.
+- **threat-service**: A threat service that exports threat events and affected Kubernetes resources, which are visible on the Cloudanix dashboard.
 
 
 
 From the **Workloads** page, click the **Risks** tab to view a list of failed threat rules. You can exclude resources, such as pods and containers, from the risk findings.
 
+<br />
+
+### Use Cloudanix with Kubernetes 1.25 and higher
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-cloudanix.md:81]:
```suggestion
### Kubernetes 1.25 and higher
```
We should shorten this. How is this suggestion?

File: content/docs/06-integrations/00-cloudanix.md [hunk 4]
```diff
@@ -70,12 +70,42 @@ This Helm Chart installs four Cloudanix services to enable container security ca
...
+
+Use the following information to find the **Spectro Namespace Labeler** add-on pack.
+
+- **Pack Type**: System App
+- **Registry**: Public Repo
+- **Pack Name**: Spectro Namespace Labeler
+- **Pack Version**: 1.0.x or higher
+
+
+Below is the YAML file.
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-cloudanix.md:93]: Do we need to show this YAML? It could lead to confusion.

Comment (ritawatson) [reply content/docs/06-integrations/00-cloudanix.md:93]: @karl-cardenas-coding I'll ask @ajinkya1986

**Comment by spectro-prow** [comment]
@karl-cardenas-coding: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 839e4f38bddab0540e6591ef0e97df8529471db5 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1276/librarium_build/1654595048366411776) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Review by ritawatson — APPROVED** [review]
LGTM

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1276#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1276#pullrequestreview-1415490692" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**